doc/Architecture.md

```diff
@@ -0,0 +1,97 @@
+﻿# Software Architecture
+
+## Context
+
+_[Insert context description here]_
```

Comment:
is this supposed to be here ?

Reply:
is it just a template for a documentation since this is the app template ?

Reply:
Yes. For generated applications (If I understand correctly).